extensions/sql-database-projects/src/sqldbproj.d.ts

```diff
@@ -263,4 +291,36 @@ declare module 'sqldbproj' {
 		sqlDW = 'Azure Synapse Dedicated SQL Pool',
 		sqlEdge = 'Azure SQL Edge'
 	}
+	export interface ISqlConnectionProperties {
+		tenantId?: string,
+		accountId?: string
+		serverName: string,
+		userName: string,
+		password: string,
+		port: number,
+		dbName: string,
+		profileName?: string,
+		connectionRetryTimeout?: number
+	}
+
+	export interface ILocalDbSetting extends ISqlConnectionProperties {
```

Comment:
maybe this one should also be changed to have Docker in the name since these additional properties are only for docker?